src/shared/use-app-shared-deps.hook.js

```diff
@@ -6,7 +6,7 @@ export const getAppDependencies = ({ name, url }) => {
   return fetch(fetchUrl)
     .then((resp) => resp.text())
     .then((jsBundleStringified) => {
-      const systemjsModule = /^System\.register\((\[.+?\])/;
+      const systemjsModule = /^System\.register\((\[.*\])/;
```

Comment:
You can test this regex out at https://regex101.com/. Try the following regex test string:

```js
System.register([], function () {
  const hi = [];
})
```